pkg/app/app.go

```diff
@@ -50,37 +51,37 @@ func New(fileName string, win *sdl.Window, cfg *config.Config) *Application {
 		W: cfg.ScreenWidth,
 		H: cfg.BottomBarHeight,
 	}
-	buttonAreaCenter := &sdl.Rect{
-		X: 0,
-		Y: 30,
-		W: 125,
-		H: 20,
-	}
 
 	bottomBarComms := make(chan comms.Image)
 	toolComms := make(chan image.Tool)
 	actionComms := make(chan func())
 
-	iv, err := image.NewView(imageViewArea, fileName, bottomBarComms, toolComms, cfg)
+	iv, err := image.NewView(imageViewArea, bottomBarComms, toolComms, cfg)
 	if err != nil {
 		log.Fatal(err)
 	}
-	bottomBar, err := NewBottomBar(bottomBarArea, bottomBarComms, cfg)
+	tex, err := gfx.NewTextureFromFile(fileName)
 	if err != nil {
 		log.Fatal(err)
 	}
-	centerButton, err := menu.NewButton(buttonAreaCenter, cfg, "Center Image", func() {
-		go func() {
-			actionComms <- func() {
-				iv.CenterImage()
-			}
-		}()
-	})
+	iv.AddLayer(tex)
+
+	bottomBar, err := NewBottomBar(bottomBarArea, bottomBarComms, cfg)
 	if err != nil {
 		log.Fatal(err)
 	}
-	centerButton.SetHighlightBackgroundColor([4]float32{1.0, 0.0, 0.0, 1.0})
-	centerButton.SetDefaultTextColor([4]float32{0.0, 0.0, 1.0, 1.0})
+	// centerButton, err := menu.NewButton(buttonAreaCenter, cfg, "Center Image", func() {
+	// 	go func() {
+	// 		actionComms <- func() {
+	// 			iv.CenterView()
+	// 		}
+	// 	}()
+	// })
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+	// centerButton.SetHighlightBackgroundColor([4]float32{1.0, 0.0, 0.0, 1.0})
+	// centerButton.SetDefaultTextColor([4]float32{0.0, 0.0, 1.0, 1.0})
```

Comment:
What is this commented code snippet for? (see also line 198)

Reply:
Removed the commented code